x-pack/plugins/observability/public/components/shared/exploratory_view/header/add_to_case_action.test.tsx

```diff
@@ -106,7 +106,10 @@ describe('AddToCaseAction', function () {
     expect(core?.cases?.ui.getAllCasesSelectorModal).toHaveBeenCalledWith(
       expect.objectContaining({
         owner: ['observability'],
-        userCanCrud: true,
+        permissions: {
+          all: false,
```

Comment:
Might want to mock `useGetUserCasesPermissions` and test against returned mocked values to make sure the data stays mapped correctly.